picard/album.py

```diff
@@ -534,8 +531,12 @@ def column(self, column):
                 unsaved = self.get_num_unsaved_files()
                 if unsaved:
                     text += '; %d*' % (unsaved,)
-                text += ungettext("; %i image", "; %i images",
-                                  len(self.metadata.images)) % len(self.metadata.images)
+                if getattr(self.metadata, 'has_common_images', True):
+                    text += ungettext("; %i image", "; %i images",
+                                      len(self.metadata.images)) % len(self.metadata.images)
+                else:
+                    text += ungettext("; %i image", "; %i different images among tracks",
```

Comment:
Originally I thought it was not possible to reach that case, but left it there just in case. After a second thought, it is possible when a release doesn't have any cover art and the user adds one image only to one track. So I'll change the singular text to be "%i image not in all tracks"